pkg/db/migrations/103_is_leaf_components.sql

```diff
@@ -0,0 +1,3 @@
+-- +goose Up
+
+ALTER TABLE components ADD COLUMN is_leaf BOOL DEFAULT false;
```

Comment:
Doesn't this need to be added to seed.sql and have the 99 script updated ?

Reply:
Yeah missed that.
Updated!